### ISSUE TITLE
Suggestion: Add a dark mode to the website.

### ISSUE THREAD

**Issue by Killer-Awesome-Phantom** [Overv/VulkanTutorial#241]
Could you add a dark mode for the website as i find it a bit harder to read in light mode?

**Comment by Overv** [comment]
I don't think it would be very easy to add a dark mode to the theme, so I would suggest to use one of the [Dark Reader extensions](https://darkreader.org/) to browse the website for now.
